kombu/transport/redis.py

```diff
@@ -959,3 +959,50 @@ def on_readable(self, fileno):
     def _get_errors(self):
         """Utility to import redis-py's exceptions at runtime."""
         return get_redis_error_classes()
+
+
+class SentinelChannel(Channel):
+    """Channel with explicit Redis Sentinel knowledge.
+
+    Broker url is supposed to look like:
+
+    sentinel://0.0.0.0:26379;0.0.0.0:26380/...
```

Comment:
I don't really like the semicolon separated urls.
This doesn't play nicely with how failover is already built in Celery.
Maybe this should be implemented as a failover strategy instead of a channel?